src/abi/index.ts

```diff
@@ -3,7 +3,13 @@ export * from './balancerQueries';
 export * from './balancerRelayer';
 export * from './balancerRouter';
 export * from './batchRelayerLibrary';
+export * from './composableStableFactoryV5.V2';
+export * from './composableStablePoolV5.V2';
 export * from './erc20';
 export * from './vaultV2';
 export * from './vaultV3';
-export * from './weightedFactoryV4';
+export * from './vaultV3Extension';
```

Comment:
If you look at vaultV3 abi, you'll see that it's already taking into account the extension. We should always access the extension through the main vault, so exposing this abi separately is not useful and my generate confusion.
```suggestion
```